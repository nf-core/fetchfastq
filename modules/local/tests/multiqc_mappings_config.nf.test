nextflow_process {

    name "Test Process MULTIQC_MAPPINGS_CONFIG"
    script "modules/local/multiqc_mappings_config.nf"
    process "MULTIQC_MAPPINGS_CONFIG"
    tag "multiqc_mappings_config"
    tag "modules_local"

    test("SRX9626017_SRR13191702") {

        when {
            params {
                outdir   = "$outputDir"
            }
            process {
                """
                input[0] = file("$baseDir/modules/local/tests/input/multiqc_mappings_config/SRX9626017_SRR13191702.mappings.csv", checkIfExists: true)
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out).match() },
            )
        }

    }

}
