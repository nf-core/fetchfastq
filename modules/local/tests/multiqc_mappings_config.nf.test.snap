{
    "SRX9626017_SRR13191702": {
        "content": [
            {
                "0": [
                    [
                        "multiqc_config.yml:md5,7f3cb10fff83ba9eb3e8fa6862d1290a",
                        "versions.yml:md5,dd4c66f0551d15510b36bb2e2b2fdd73"
                    ]
                ],
                "1": [
                    "versions.yml:md5,dd4c66f0551d15510b36bb2e2b2fdd73"
                ],
                "versions": [
                    "versions.yml:md5,dd4c66f0551d15510b36bb2e2b2fdd73"
                ],
                "yml": [
                    [
                        "multiqc_config.yml:md5,7f3cb10fff83ba9eb3e8fa6862d1290a",
                        "versions.yml:md5,dd4c66f0551d15510b36bb2e2b2fdd73"
                    ]
                ]
            }
        ],
        "timestamp": "2023-07-05T15:01:18+0000"
    }
}